soh/src/overlays/actors/ovl_En_Ds/z_en_ds.c

```diff
@@ -55,12 +55,12 @@ void EnDs_Destroy(Actor* thisx, PlayState* play) {
     ResourceMgr_UnregisterSkeleton(&this->skelAnime);
 }
 
-void EnDs_Talk(EnDs* this, PlayState* play) {
-    if (Actor_TextboxIsClosing(&this->actor, play)) {
-        this->actionFunc = EnDs_Wait;
-        this->actor.flags &= ~ACTOR_FLAG_WILL_TALK;
+void EnDs_Talk(EnDs* that, PlayState* play) {
+    if (Actor_TextboxIsClosing(&that->actor, play)) {
+        that->actionFunc = EnDs_Wait;
+        that->actor.flags &= ~ACTOR_FLAG_WILL_TALK;
     }
-    this->unk_1E8 |= 1;
+    that->unk_1E8 |= 1;
 }
```

Comment:
Any reason for this change?

Reply:
In order to make my VB hooks work, I declared these functions in the header to be in scope for the hook handler. When I tried to build like this however I got a very odd exception `expected ‘,’ or ‘...’ before ‘this’` on the deceleration I found no help on. Changing "this" to "that" resolves this exception, which I assume is related to "this" having specific meaning in C. I will gladly accept a better solution.

Reply:
The argument names do not need to match across header & c files, so you can leave this file alone and have it as `that` (or `enDs` or `thisx` or `actor`) in the header file

Reply:
Didn't know, as VSCode was saying it couldn't find it i assumed it had to be the same, changed it.